RELEASE_NOTES.md

```diff
@@ -622,15 +622,15 @@ Notable changes include:
       * New OpenMP execution policies added to support static, dynamic, and 
         guided scheduling.
       * Added support for const iterators to be used with RAJA scans.
-      * Support for bitwise and and or reductions have been added.
+      * Support for bitwise and or reductions have been added.
```

Comment:
```suggestion
      * Support for bitwise "and" and "or" reductions have been added.
```

Double quotes in markdown files are rendered properly when you open a markdown file in GitHub and also in the way it appears in a browser.